### DISCUSSION TITLE
server on the list

### DISCUSSION THREAD

**Discussion by mastodonLike** [mastodon/mastodon#20224] [General]
Hello, when my server will be listed among the other server on the front page of mastodon, please?

**Comment by YJSoft** [comment]
https://joinmastodon.org/covenant

## Submitting your server
If you are a server owner, and your server passes these requirements, you can send an e-mail to hello@joinmastodon.org with the subject "Server submission" to be listed on this website. Because our goal is giving new users a smooth and reliable start into the decentralized social network of Mastodon rather than simply promoting individual servers, we are generally not interested in promoting brand new and unproven servers, so the submission should not be your primary strategy of growing your community.

**Reply by mastodonLike** [reply]
thank you

**Comment by redegelde** [comment]
hi, wrote three mail to the known mail adres but no result
Education is not present
as we speak the instance https://mastodon.education is running smooth and stable>
Do i mis something to get present at the server listing

Regard and keep up the growing work
Eric
